SeniorInterview/StocksAnalyzed/src/Requestor.java

```diff
@@ -1,3 +1,23 @@
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+
 public class Requestor {
```

Comment:
Might be interesting to have an Interface for this, as well as have this as an Abstract class